packages/appkit/src/client.ts

```diff
@@ -1605,7 +1609,8 @@ export class AppKit {
   }: Pick<AdapterBlueprint.ConnectResult, 'address' | 'chainId'> & {
     chainNamespace: ChainNamespace
   }) {
-    if (chainNamespace !== 'eip155') {
+    const chain = this.caipNetworks?.find(n => n.caipNetworkId === `${chainNamespace}:${chainId}`)
```

Comment:
```suggestion
    const currentChain = this.caipNetworks?.find(n => n.caipNetworkId === `${chainNamespace}:${chainId}`)
```

or `activeChain` for more readable namings